targets/csharp/source/source/Extensions/Models/PlayFabCloudScriptModelsExtensions.cs.ejs

```diff
@@ -0,0 +1,71 @@
+namespace PlayFab.CloudScriptModels
+{
+    using System.Threading.Tasks;
+    /// <summary>
+    /// Used for extracting the data fields incoming as payload to Azure Functions into their respective objects. 
+    /// Provides  fields through ApiSettings and AuthenticationContext, the profile of the caller, as well as the 
+    /// arguments passed to the execution request.
+    /// </summary>
+    /// <typeparam name="TFunctionArgument">Type of FunctionArgument</typeparam>
+    public class FunctionExecutionContext<TFunctionArgument>
+    {
+        public PlayFabApiSettings ApiSettings { get; private set; }
+
+        public PlayFabAuthenticationContext AuthenticationContext { get; set; }
+
+        public ProfilesModels.EntityProfileBody CallerEntityProfile { get; set; }
+
+        public TFunctionArgument FunctionArgument { get; set; }
+
+        protected FunctionExecutionContext() { }
+
+        /// <summary>
+        /// Creates a new <c>FunctionExecutionContext</c> out of the incoming request to an Azure Function.
+        /// </summary>
+        /// <param name="request">The request incoming to an Azure Function from the PlayFab server</param>
+        /// <returns>A new populated <c>FunctionExecutionContext</c> instance</returns>
+        public static async Task<FunctionExecutionContext<TFunctionArgument>> Create(System.Net.Http.HttpRequestMessage request)
```

Comment:
Can this also be moved to a using statement?

Reply:
Moved `Content` to using.